VM-RIght-Sizing/manifest.yaml

```diff
@@ -0,0 +1,29 @@
+manifestVersion: 1.0
+projects:
+  - name: jira-connection
+    path: first-example
+  - name: slack-connection
+    path: second-example
+  - name: workflow-calculate-ticket
+    path: third-example
+
+environmentGroups:
+  - name: production
+    environments:
+      - name: production-environment
+        url:
+          value: "" # Enter environment ID e.g. https://<tenantId>.apps.dynatrace.com/
+        auth:
+          token:
+            type: environment
+            name: DT_API_TOKEN
+          oAuth:
+            clientId:
+              type: environment
+              name: DT_OAUTH_CLIENT_ID
+            clientSecret:
+              type: environment
+              name: DT_OAUTH_CLIENT_SECRET
+            tokenEndpoint:
```

Comment:
The `tokenEndpoint` isn't needed for production environments (defaults to `https://sso.dynatrace.com/sso/oauth2/token`) and is generally just used against different stages of SSO internally. 

I would remove this from the public sample.

Reply:
Removed the token endpoint as well as the env variable in input-needed.sh.